torch/ao/quantization/fake_quantize.py

```diff
@@ -353,12 +356,8 @@ def forward(self, X: torch.Tensor) -> torch.Tensor:
 Default dynamic fake_quant for activations.
 """
 
-# TODO(future PR): remove these defaults and enforce activation functions
-# to explicitly specify their output range
-default_symmetric_fixed_qparams_fake_quant = FixedQParamsFakeQuantize.with_args(
-    scale=2.0 / 256.0, zero_point=128, dtype=torch.quint8, quant_min=0, quant_max=255)
-default_affine_fixed_qparams_fake_quant = FixedQParamsFakeQuantize.with_args(
-    scale=1.0 / 256.0, zero_point=0, dtype=torch.quint8, quant_min=0, quant_max=255)
+default_symmetric_fixed_qparams_fake_quant = FixedQParamsFakeQuantize.with_args(**default_symmetric_fixed_qparams_args)
```

Comment:
wondering if we can use the default FakeQuantize + FixedQParamObserver now to represent this